Extending/Key-Vault/index.md

```diff
@@ -1,14 +1,19 @@
 ---
 versionFrom: 10.0.0
 meta.Title: "Azure Key Vault"
-meta.Description: "A guide configuring an Azure Key Vault"
+meta.Description: "A guide for configuring Azure Key Vault"
 ---
 
-# Configuring an Azure Key Vault
+# Configuring Azure Key Vault
 
-From a security perspective, it's always a good solution to store your application secrets (such as a connection string and others keys) in an Azure Key Vault. This article tells you how to configure your application so that it's ready to use a Key Vault
+From a security perspective, it's always a good solution to store your application secrets (such as a connection string and others keys) in Azure Key Vault. This article tells you how to configure your application so that it's ready to use a Key Vault.
 
-## Installing the packages
+There are a few approaches to incorporate Azure Key Vault into your application, depending on your hosting situation.
+
+1. Install Key Vault via Nuget and retrieve your secrets directly in your Program.cs. (For most Hosting scenarios)
+2. Directly reference a secret in your Azure Web App Configuration Application Settings. (For Azure Web App Hosting)
```

Comment:
How about using this list as a table of content and link down to the related sections in the article?

Reply:
Could you give me another page on Our that does this? Not sure what the syntax would look like in Markdown.

Reply:
Done